src/cascadia/LocalTests_TerminalApp/CommandlineTest.cpp

```diff
@@ -799,4 +783,67 @@ namespace TerminalAppLocalTests
         VERIFY_ARE_EQUAL(ShortcutAction::SplitPane, appArgs._startupActions.at(1).Action());
         VERIFY_ARE_EQUAL(ShortcutAction::SplitPane, appArgs._startupActions.at(2).Action());
     }
+
+    void CommandlineTest::CheckTypos()
+    {
+        Log::Comment(NoThrowString().Format(
+            L"Check what happens when the user typo's a subcommand. It should "

```

Comment:
I can't suggest in filtered view. This is an indicator for me to come back and add a suggest for `typo's` to `typos`

Reply:
```suggestion
            L"Check what happens when the user typos a subcommand. It should "
```

Reply:
![image](https://user-images.githubusercontent.com/18356694/72080831-0c06b880-32c3-11ea-932f-347e6465fa2c.png)